README.md

```diff
@@ -3,84 +3,52 @@
 ### Egg party is a Slack App for thanking teammates, raising chickens, and having a rowdy ol' time
 
 ---
+
 #### Table of Contents
-- [Egg Party 🥚🎉](#egg-party-%f0%9f%a5%9a%f0%9f%8e%89)
-    - [Egg party is a Slack App for thanking teammates, raising chickens, and having a rowdy ol' time](#egg-party-is-a-slack-app-for-thanking-teammates-raising-chickens-and-having-a-rowdy-ol-time)
-      - [Table of Contents](#table-of-contents)
-  - [* Helpful Links](#helpful-links)
-    - [**Want to contribute?**](#want-to-contribute)
-      - [**Requirements:**](#requirements)
-      - [**First-Time Setup:**](#first-time-setup)
-        - [DB Setup with MS SQL](#db-setup-with-ms-sql)
-          - [DB Setup Troubleshooting Notes](#db-setup-troubleshooting-notes)
-        - [Slack Setup](#slack-setup)
-      - [**Local Configuration Files**](#local-configuration-files)
-    - [**Running & Debugging (VS Code)**](#running--debugging-vs-code)
-        - [Note: You'll need to change the Events Request URL and Interactions Request URL every time you start a new ngrok session.](#note-youll-need-to-change-the-events-request-url-and-interactions-request-url-every-time-you-start-a-new-ngrok-session)
-    - [**Helpful Links:**](#helpful-links)
+- [**Want to contribute?**](#want-to-contribute)
+- [**Local Setup**](#local-setup)
+    - [DB Setup](#db-setup-with-ms-sql)
```

Comment:
I'm guessing you can remove this line since it doesn't navigate anywhere anymore